File: 251025/수의 계절은/season-of-num.c
#include <stdio.h>

int main() {
    // Please write your code here.
    int m;
    scanf("%d", &m);

    if (3 <= m && m <= 5) {
        printf("Spring");
    }else if (6 <= m && m <= 8) {
        printf("Summer");
    }else if (9 <= m && m <= 11) {
        printf("Fall");
    }else {
        printf("Winter");
    }
    return 0;
}

File: 251119/수의 계절은/season-of-num.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int m = sc.nextInt();

        if (3 <= m && m <= 5) {
            System.out.printf("Spring");
        }else if (6 <= m && m <= 8) {
            System.out.printf("Summer");
        }else if (9 <= m && m <= 11) {
            System.out.printf("Fall");
        }else {
            System.out.printf("Winter");
        }
    }
}